pkgs/by-name/do/dotcopy/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchzip
+}:
+
+buildGoModule rec {
+  pname = "dotcopy";
+  version = "0.2.14";
+
+  src = fetchzip {
+    url = "https://github.com/FireSquid6/dotcopy/archive/refs/tags/v${version}.zip";
+    hash = "sha256-oVMsIZUJ7xOBwSlJF+RUIYG0dPMTZ3ftDd9cpRytl7w=";
+  };
+
+  doCheck = false;
+  vendorSha256 = null;
+
+  subPackages = [ "." ];
+
+  meta = with lib; {
+    description = "A linux dotfile manager";
+    homepage = "https://dotcopy.firesquid.co";
```

Comment:
404

Reply:
it won't be in a few days so I figured I'd go ahead and set it up so I didn't have to waste someone's time with another PR

Reply:
```suggestion
    homepage = "https://dotcopy.firesquid.com/";
```